src/ParseDS.java

```diff
@@ -42,11 +44,14 @@ public void buildMessages() {
     // Given Message type, return an array of the fields
     public ArrayList<Object> getFields(String mType) {
         Map<Object, Object> tempMap = (Map<Object, Object>) mMap.get(mType);
+	assert tempMap != null : "File type missing: " + mType;
```

Comment:
@Mister-Meeseeks I never use asserts in code. Only in tests. debugger outputs are best. they can be categorized into log, warn, debug etc.